#
# old_revision [3189858dee662e616a3978601d74269683a55f48]
#
# patch "file2"
#  from [4f8091d2fa31b0598dd0ff6dc936f8d0b47b59d1]
#    to [8aa9b19b304a24ebcd71cfc4a27efc95f8ad8e11]
#
============================================================
--- file2	4f8091d2fa31b0598dd0ff6dc936f8d0b47b59d1
+++ file2	8aa9b19b304a24ebcd71cfc4a27efc95f8ad8e11
@@ -1,3 +1,4 @@ bar
 foo
 bar
-quux
\ No newline at end of file
+quux
+baz
